live-examples/js-examples/typedarray-set.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js" class="language-js">// create a SharedArrayBuffer with a size in bytes
+var buffer = new ArrayBuffer(8);
+var uint8 = new Uint8Array(buffer);
+
+uint8.set([1, 2, 3], 3);
```

Comment:
Nit: This could perhaps benefit from a comment like "Copy the values given, starting at index 3", or something better than that.